src/coreclr/vm/gcenv.ee.cpp

```diff
@@ -444,6 +444,31 @@ gc_alloc_context * GCToEEInterface::GetAllocContext()
     return pThread->GetAllocContext();
 }
 
+void InvokeGCAllocCallback(ee_alloc_context* pEEAllocContext, enum_alloc_context_func* fn, void* param)
+{
+    // the allocation context can be null initially, so we need to take that case into account: skip the threshold management
```

Comment:
Do you mean that alloc_ptr can be NULL? If pAllocContext is NULL this code will crash.

Also its not clear what the comment means when it says we skip threshold management. I think the comment is reffering to past version of this code. 
```suggestion
    // NOTE: Its possible that alloc_ptr = alloc_limit = alloc_sampling = NULL at this point.
```

Reply:
Good catch: it refers to the previous implementation
I'm updating with your proposal where the fields of the allocation context could be equally NULL